deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -3985,8 +3986,8 @@ NAN_METHOD(WebGLRenderingContext::GetVertexAttribOffset) {
 }
 
 NAN_METHOD(WebGLRenderingContext::GetShaderPrecisionFormat) {
-  GLenum shaderType = info[0]->Uint32Value();
-  GLenum precisionType = info[1]->Uint32Value();
+  GLenum shaderType = TO_UINT32(info[0]);
```

Comment:
should these be `TO_INT32`? (elsewhere you use `TO_INT32` for casting to `GLenum`)

Reply:
Yes, thank you.